experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.HandoffBot/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
```

Comment:
I don't think our samples typically have this file.

Reply:
These are settings that specify how to run the bot locally. This came from the bot template.

Reply:
not changed